app/root.css

```diff
@@ -160,6 +160,9 @@ h2 {
   letter-spacing: -0.14px;
 }
 
+.leading-0 {
+  line-height: 0 !important;
```

Comment:
@melissasamworth

Reply:
When I was trying to style the button, It kept putting this annoying space below it. After some testing, I discovered it was a line-height issue. I've added this to the root incase there are other circumstances where there is a line-height problem.

Reply:
Fine with me

Reply:
Except sometimes the button actually goes to two lines. In which case this would end up looking really weird and bad obviously.